src/apps/frontend/pages/authentication/login/login-form.tsx

```diff
@@ -23,6 +25,15 @@ interface LoginFormProps {
 
 const LoginForm: React.FC<LoginFormProps> = ({ onError, onSuccess }) => {
   const { formik, isLoginLoading } = useLoginForm({ onSuccess, onError });
+  const currentLoginMethod = Config.getConfigValue<string>(
+    'authenticationMechanism'
+  );
+  const navigate = useNavigate();
+  useEffect(() => {
+    if (currentLoginMethod === constant.PHONE_NUMBER_BASED_AUTHENTICATION) {
+      navigate(routes.SIGNUP);
```

Comment:
It is not clear why ar we redirecting the user to signup page when user come to login page for phone base auth.